fhi_users/auth/rest_auth_views.py

```
@@ -29,6 +29,7 @@
 from django.views.decorators.vary import vary_on_cookie
 from django.utils.decorators import method_decorator
 
+from loguru import logger
```

Ruff (0.8.2)

Redefinition of unused `logger` from line 1

Remove definition: `logger`

(F811)